src/EleventyFiles.js

```diff
@@ -329,14 +329,14 @@ class EleventyFiles {
 		return ret;
 	}
 
-	_globSearch() {
+	async _globSearch() {
 		let globs = this.getFileGlobs();
 
 		// returns a promise
 		debug("Searching for: %o", globs);
-		return this.fileSystemSearch.search("templates", globs, {
+		return (await this.fileSystemSearch.search("templates", globs, {
 			ignore: this.uniqueIgnores,
-		});
+		})).map(i => i.replace(/\\/g, '/'));
```

Comment:
Using [`path.sep`](https://nodejs.org/api/path.html#pathsep) would work here, too?

Instead of an IIFE for the promise I think it's okay to add another temporary variable.

```suggestion
		const results = await this.fileSystemSearch.search("templates", globs, {
			ignore: this.uniqueIgnores,
		});
		return results.map(i => i.replace(new RegExp(path.sep, 'g'), '/'));
```

Untested, as I'm on Debian Linux.

Reply:
I'll add the temporary variable for clarity, but `path.sep` would not work here without being escaped. I don't think there'd be any benefit in using it either: replacing linux path separators would not do anything.